gradle/libs.versions.toml

```diff
@@ -0,0 +1,156 @@
+opentelemetry-exporter-otlp = { module = "io.opentelemetry:opentelemetry-exporter-otlp", version.ref = "opentelemetry" }
+opentelemetry-sdk = { module = "io.opentelemetry:opentelemetry-sdk", version.ref = "opentelemetry" }
+opentelemetry-sdk-testing = { module = "io.opentelemetry:opentelemetry-sdk-testing", version.ref = "opentelemetry" }
+opentelemetry-semconv = { module = "io.opentelemetry.semconv:opentelemetry-semconv", version.ref = "opentelemetry-semconv" }
+
+disruptor = { module = "com.lmax:disruptor", version.ref = "disruptor" }
+
+bouncycastle-certs = { module = "org.bouncycastle:bcpkix-jdk18on", version.ref = "bouncycastle" }
+bouncycastle-provider = { module = "org.bouncycastle:bcprov-jdk18on", version.ref = "bouncycastle" }
+
+mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
+mockito-junit-jupiter = { module = "org.mockito:mockito-junit-jupiter", version.ref = "mockito" }
+
+hamcrest = { module = "org.hamcrest:hamcrest", version.ref = "hamcrest" }
+
+jmh-core = { module = "org.openjdk.jmh:jmh-core", version.ref = "jmh-core" }
+
+semver4j = { module = "org.semver4j:semver4j", version.ref = "semver4j" }
+
+json = { module = "org.json:json", version.ref = "json" }
+
+polyglot = { module = "org.graalvm.polyglot:polyglot", version.ref = "graalvm" }
+polyglot-js-community = { module = "org.graalvm.polyglot:js-community", version.ref = "graalvm" }
+
+toxiproxy = {module = "eu.rekawek.toxiproxy:toxiproxy-java", version.ref = "toxiproxy" }
+
+testcontainers = {module = "org.testcontainers:testcontainers", version.ref = "testcontainers" }
+testcontainers-kafka = {module = "org.testcontainers:kafka", version.ref = "testcontainers" }
+testcontainers-junit-jupiter = {module = "org.testcontainers:junit-jupiter", version.ref = "testcontainers" }
+testcontainers-opensearch = {module = "org.opensearch:opensearch-testcontainers", version.ref = "opensearch-testcontainers" }
+testcontainers-toxiproxy = {module = "org.testcontainers:toxiproxy", version.ref = "testcontainers" }
+
+reactor-test = {module = "io.projectreactor:reactor-test", version.ref = "reactor-test" }
+reactor-netty-core = {module = "io.projectreactor.netty:reactor-netty-core", version.ref = "reactor-netty" }
+reactor-netty-http = {module = "io.projectreactor.netty:reactor-netty-http", version.ref = "reactor-netty" }
+
+lucene-v7-core = { module = "org.apache.lucene:lucene-core", version.ref = "lucene7" }
+lucene-v7-backward-codecs = { module = "org.apache.lucene:lucene-backward-codecs", version.ref = "lucene7" }
+
+lucene-v9-core = { module = "org.apache.lucene:lucene-core", version.ref = "lucene9" }
+lucene-v9-analysis-common = { module = "org.apache.lucene:lucene-analysis-common", version.ref = "lucene9" }
+lucene-v9-backward-codecs = { module = "org.apache.lucene:lucene-backward-codecs", version.ref = "lucene9" }
+
+jni-java = { module = "com.hubspot.jinjava:jinjava", version.ref = "jni-java" }
```

Comment:
let's fix this one to jinjava - jni and jin are very different things.  The hyphen also breaks the 'jinja' part

Reply:
Thank you - this dyslexic developer sees nothing wrong :D